### ISSUE TITLE
【M5Atom Matrix】The initial screen is not reflected in the code.

### ISSUE THREAD

**Issue by mongonta0716** [m5stack/UiFlow-Bug-Report#47]
I made the screen settings on the left side of UIFlow, but they are not reflected in the code, so when I download or reload the code to M5Atom Matrix, it does not run.

Normally, the letter "X" should appear on the initial screen, but nothing is displayed.

![UIFlow](https://user-images.githubusercontent.com/24690724/79042577-e677ab00-7c33-11ea-9479-fd10e8a908ec.PNG)

```
from m5stack import *
from m5ui import *
from uiflow import *

rgb.set_screen([0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0])





def buttonA_wasPressed():
  # global params
  rgb.set_screen([0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00])
  pass
btnA.wasPressed(buttonA_wasPressed)

def buttonA_wasReleased():
  # global params
  rgb.set_screen([0xff0000,0xff0000,0x00ff00,0xff0000,0xff0000,0xff0000,0x00ff00,0x00ff00,0x00ff00,0xff0000,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0x00ff00,0xff0000,0x00ff00,0x00ff00,0x00ff00,0xff0000,0xff0000,0xff0000,0x00ff00,0xff0000,0xff0000])
  pass
btnA.wasReleased(buttonA_wasReleased)


rgb.setBrightness(10)

```

**Comment by EeeeBin** [comment]
Please try to use the mouse to click on the atom color block, do not drag

**Comment by mongonta0716** [comment]
Do I have to repeat every time I read an m5f file?

**Comment by EeeeBin** [comment]
No need, This problem only occurs if the mouse is dragged out of the atom screen

**Comment by mongonta0716** [comment]
That means it's a bug that the initial screen may not be saved in M5F.

```
{"components":[{"id":"____screen","createTime":1586606058425,"name":"Screen","x":0,"y":0,"width":320,"height":240,"backgroundColor":"#222222","backgroundImage":"","type":"screen"}],"type":"atom-matrix","versions":"Stabile","units":[],"hats":[],"blockly":"<block type=\"basic_on_setup\" id=\"setup_block\" deletable=\"false\" x=\"50\" y=\"50\"><next><block type=\"rgb_set_brightness\" id=\"Yi/*UJqqjX-=2?:mJ(YE\"><value name=\"BRIGHTNESS\"><shadow type=\"math_number\" id=\"FQIoYpe=I5Wi[nYbmO;4\"><field name=\"NUM\">10</field></shadow></value></block></next></block><block type=\"button_callback\" id=\"f19egWOi7*Qd(h{u=n9k\" x=\"50\" y=\"170\"><field name=\"BUTTON\">A</field><field name=\"EVENT\">wasPressed</field><statement name=\"FUNC\"><block type=\"rgb_set_led_matrix\" id=\"/P_ev+u:4E[^-W~?c$-L\"><field name=\"COLOUR\">#00ff00</field><field name=\"MATRIX\" matrix=\"#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00\"></field></block></statement></block><block type=\"button_callback\" id=\"iU;J~}MT+Coig7=;t$]O\" x=\"50\" y=\"450\"><field name=\"BUTTON\">A</field><field name=\"EVENT\">wasReleased</field><statement name=\"FUNC\"><block type=\"rgb_set_led_matrix\" id=\"ub[ga6{p`:r!mYNt)#VZ\"><field name=\"COLOUR\">#00ff00</field><field name=\"MATRIX\" matrix=\"#ff0000,#ff0000,#00ff00,#ff0000,#ff0000,#ff0000,#00ff00,#00ff00,#00ff00,#ff0000,#00ff00,#00ff00,#00ff00,#00ff00,#00ff00,#ff0000,#00ff00,#00ff00,#00ff00,#ff0000,#ff0000,#ff0000,#00ff00,#ff0000,#ff0000\"></field></block></statement></block>","Blockly.Remotes":[],"modules":["rgb"],"matrix":["#ff0000","#0000ff","#0000ff","#0000ff","#00ff00","#0000ff","#ff0000","#0000ff","#00ff00","#0000ff","#0000ff","#0000ff","#00ff00","#0000ff","#0000ff","#0000ff","#00ff00","#0000ff","#ff0000","#0000ff","#00ff00","#0000ff","#0000ff","#0000ff","#ff0000"]}
```

**Comment by mongonta0716** [comment]
Keep in mind that the phenomenon is reproducible so you don't have to fix it right away.

**Comment by mongonta0716** [comment]
I've seen it fixed in v1.5.3, so I'll close it.
